src/lib/components/molecules/column-chart/column-chart-util.js

```diff
@@ -0,0 +1,12 @@
+export function scaleLinear(domain, range) {
```

Comment:
do we need this file in the end? looks like it's not being imported and we're using d3-scale in `src/lib/components/molecules/column-chart/index.jsx` ?

Reply:
this is being used elsewhere in the library. I'd like to revisit whether it's tidier to use this than d3 as I suspect it might be. but in any event I don't think we can delete it.